1-mandatory/1-writers.js

```diff
@@ -62,6 +65,12 @@ Exercise 2:
 */
 
```

Comment:
A couple of small points on this one.
1. Maybe it's just a typo, but it's a good idea to use `===` instead of `==`. Can you remember what the difference is between them 😄? Especially in larger code bases, using `===` will help you write more predictable code with fewer bugs.
2. For practice, you can try splitting this out into a few array methods: maybe `filter` first, and then `forEach`? You could also chain these array methods together.